torch_xla/experimental/distributed_checkpoint.py

```diff
@@ -214,6 +304,52 @@ def commit_tensor(self, read_item: ReadItem, tensor: torch.Tensor) -> None:
       self.sharded_state_dict[fqn].load_local_shards_(local_shards)
 
 
+def _create_write_item_from_indices(fqn: str, shard_index: int,
```

Comment:
Just noticed that you created all the helpers as the private functions of the module instead of private methods within each class. Just curious on why? You don't need to change this.

Reply:
Ah no real reason, I was just following the pattern from the upstream implementation - helpers to generate Read/WriteItems are private in the module for the DefaultPlanners.

Reply:
I still won't call myself a Python expert and that's why I ask questions on python programming patterns when they contradict to my C++ instinct.